packages/bpk-component-mobile-scroll-container/src/BpkMobileScrollContainer.tsx

```diff
@@ -70,51 +67,30 @@ const computeScrollIndicatorClassName = (
 };
 
 type Props = {
-  children: Node,
-  innerContainerTagName: string,
-  className: ?string,
-  leadingIndicatorClassName: ?string,
-  trailingIndicatorClassName: ?string,
-  scrollerRef: ?Function,
-  style: ?Object,
-  showScrollbar: ?boolean,
+  ariaLabel?: string | null;
```

Comment:
```suggestion
  ariaLabel?: string;
```

Reply:
Hi @olliecurtis Thank you for the review. I've removed `null` props here and updated related tests.